.travis.yml

```diff
@@ -1,13 +1,15 @@
-sudo: required
-dist: trusty
```

Comment:
> use the more recent and faster Linux virtualized infrastructure (rather than the older containerized infrastructure) by switching Linux testing to Ubuntu Trusty

We already switched to Ubuntu Trusty since e2966a4f8.

Now about the change of infrastructure it doesn't seem effective.
Before this change:
```
Worker information
hostname: d34354d8-c92b-4ea1-abfc-be17633445f7@1.production-1-worker-org-gce-x71t
version: v6.1.0 https://github.com/travis-ci/worker/tree/a6071dd64658ecf13ec3190bfb9e9ca3c8f3e10d
instance: travis-job-f9799a07-703b-45ee-9ff2-a4cfcd2888cc travis-ci-garnet-trusty-1512502259-986baf0 (via amqp)

Operating System Details
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.5 LTS
Release:	14.04
Codename:	trusty
```
(gce is "google cloud engine")

After this change:
```
Worker information
hostname: 9923f557-b415-4c62-a739-50504b2abf4b@1.production-1-worker-org-gce-s54k
version: v6.1.0 https://github.com/travis-ci/worker/tree/a6071dd64658ecf13ec3190bfb9e9ca3c8f3e10d
instance: travis-job-8ce90d97-dbda-4dc1-8a51-8b9b0555602a travis-ci-garnet-trusty-1512502259-986baf0 (via amqp)

Operating System Details
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.5 LTS
Release:	14.04
Codename:	trusty
```

So maybe the change is OK, but we're in fact just removing "dead configuration"
which isn't necessary anymore?